src/statsD.d.ts

```diff
@@ -0,0 +1,42 @@
+import MetricsClient from './MetricsClient';
```

Comment:
Do we still need this even though we're not re-exporting the type anymore?

Reply:
Strictly speaking no, this only served as type safety theatre within `createStatsDClient` itself. I didn't realise TypeScript would erase that but it does 🎉